src/test/java/org/mskcc/cbio/oncokb/web/rest/AccountResourceIT.java

```diff
@@ -58,6 +62,24 @@ public class AccountResourceIT {
     @Autowired
     private MockMvc restAccountMockMvc;
 
+    @Autowired
+    private ApplicationProperties applicationProperties;
+
+    RecaptchaProperties recaptchaProp;
+
+    private AccountResource accountResource;
+
+    @BeforeEach
+    public void setUp() throws IOException {
+        recaptchaProp = new RecaptchaProperties();
+        recaptchaProp.setProjectId("symbolic-nation-320615");
+        recaptchaProp.setSiteKey("6LfAOe4jAAAAANjzxWQ8mKilcvk1QvLLohd7EV7F");
```

Comment:
The `projectId` and `siteKey` would be needed in the AccountResourceIT since the recaptcha client checks for these properties. The site key is a public key